packages/plugin-team/test/integration/spec/actions.js

```diff
@@ -5,44 +5,520 @@
  */
 
 import '../..';
+import '@ciscospark/plugin-wdm';
 
-describe(`plugin-team`, function() {
-  describe(`#addConversation()`, () => {
-    it(`adds an existing group conversation to a team`, () => {});
-  });
+import {assert} from '@ciscospark/test-helper-chai';
+import CiscoSpark from '@ciscospark/spark-core';
+import {find} from 'lodash';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
 
-  describe(`#addMember()`, () => {
-    it(`adds a team member to a team`, () => {});
-  });
+describe(`plugin-team`, () => {
+  describe(`Team`, () => {
 
-  describe(`#assignModerator()`, () => {
-    it(`assigns a team moderator`, () => {});
-  });
+    let kirk, spock;
 
-  describe(`#archive()`, () => {
-    it(`archives a team`, () => {});
-    it(`archives a team conversation`, () => {});
-  });
+    before(() => testUsers.create({count: 2})
+      .then((users) => {
+        [kirk, spock] = users;
 
-  describe(`#joinConversation()`, () => {
-    it(`adds the user to an open team conversation`, () => {});
-  });
+        kirk.spark = new CiscoSpark({
+          credentials: {
+            authorization: kirk.token
+          },
+          config: {
+            conversation: {
+              keepEncryptedProperties: true
+            }
+          }
+        });
 
-  describe(`#removeConversation()`, () => {
-    it(`removes a team conversation from a team`, () => {});
-  });
+        spock.spark = new CiscoSpark({
+          credentials: {
+            authorization: spock.token
+          },
+          config: {
+            conversation: {
+              keepEncryptedProperties: true
+            }
+          }
+        });
 
-  describe(`#removeMember()`, () => {
-    it(`removes a team member from a team`, () => {});
-  });
+        return Promise.all([
+          kirk.spark.mercury.connect(),
+          spock.spark.mercury.connect()
+        ]);
+      })
+    );
 
-  describe(`#unassignModerator()`, () => {
-    it(`unassigns a teamMember as a moderator`, () => {});
-  });
+    after(() => Promise.all([
+      kirk.spark.mercury.disconnect(),
```

Comment:
add safety checks to make sure kirk, spock, etc, are defined; please check all your tests for this.